### PR TITLE
Fix IE svg issue in bootloader

### PR DESCRIPTION
**Pull request by AdamGMiller**
Update pre-bootloader for document app to handle svg-related issue where svgs don't properly display as a background-image css style in IE.

### CODE REVIEW

**Comment by jacquie** [comment]
The reason I put the tri-flame above was because (and this might just be me!) positioned right next to the logo it looks like it's leaning over to the left slightly.

It does work in IE ok.  Pity it can't deal with svg background images :(

**Comment by AdamGMiller** [comment]
I ended up grabbing what I knew was generally working from Measures Manager, but I have zero investment in where the TriFlame should go.  Happy to defer to Andrew and put it wherever it makes sense.

File: docs/index.html [hunk 1]
```diff
@@ -89,16 +70,14 @@
...
-        <div id="pre-bootstrap">
-            <div class="messaging">
-                <h1>
-                    <img src="./assets/TriFlame.svg" style="width:24px">
-                </h1>
-                <div class="logo-background-brand">
+                <div id="pre-bootstrap">
+                    <div class="messaging">
+                        <h1>
+                            <img src="./assets/TriFlame.svg" width="17" height="50"> <img src="./assets/CashmereAppLogo.svg" width="200" height="50">
```
Comment (andrew-frueh) [inline docs/index.html]: Thanks for doing this @AdamGMiller!  Just a couple adjustments to this line, the tri-flame should be a little bigger, and on a separate line from the logo (or else Paul will get mad that we're messing with the master brand):
`<br>`

**Comment by AdamGMiller** [comment]
Okay - changes updated.  Appreciate the feedback!

**Pull request merged**